src/libraries/oob.proj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup Condition="'$(BuildTargetFramework)' == '$(NetCoreAppCurrent)' or '$(BuildTargetFramework)' == ''">
     <ProjectReference Include="sfx-src.proj" OutputItemType="SharedFrameworkAssembly" />
+    <ProjectReference Include="$(CoreLibProject)" OutputItemType="SharedFrameworkAssembly" />
```

Comment:
This looks wrong. CoreLib isn't part of the out-of-band build.

Reply:
see above comment about the sfx trimming